www/js/lib/zimDirEntry.js

```diff
@@ -116,6 +116,15 @@ define([], function() {
     DirEntry.prototype.getTitleOrUrl = function() {
         return this.title ? this.title : this.url;
     };
+    
+    /**
+     * Looks up the dirEntry's mimetype number in the ZIM file's MIME type list, and returns the corresponding MIME type
+     * 
+     * @return {String} The MIME type corresponding to mimetype in the ZIM file's MIME type list
+     */
+    DirEntry.prototype.getMimetype = function() {
+        return this._zimfile.mimeTypes.get(this.mimetype);
```

Comment:
For the ease of understanding, what would you think of renaming the `mimetype` attribute of a DirEntry to `mimetypeInteger` or something similar?
That would prevent from some confusion between it and this `getMimetype` function, that does not return the integer, but the corresponding string

Reply:
Fine by me! Will do.